langchain/src/retrievers/parent_document.ts

```diff
@@ -129,7 +145,11 @@ export class ParentDocumentRetriever extends MultiVectorRetriever {
       embeddedDocs.push(...taggedSubDocs);
       fullDocs[parentDocId] = parentDoc;
     }
-    await this.vectorstore.addDocuments(embeddedDocs);
+    if (this.retriever) {
+      await this.retriever.addDocuments(embeddedDocs);
```

Comment:
I don't think this is a standard method

Reply:
If it's a class extending from `VectorStoreRetriever` then it will have it.